ios/FluentUI/Tooltip/TooltipView.swift

```diff
@@ -136,12 +94,141 @@ class TooltipView: UIView {
+            else {
+                return message
+            }
+
+            return title + message
+        }
+        set { }
+    }
+
+    override var accessibilityHint: String? {
+        get { return "Accessibility.Dismiss.Hint".localized }
+        set { }
     }
 
-    override func didMoveToWindow() {
-        super.didMoveToWindow()
-        updateWindowSpecificColors()
+    private func updateAppearance(tokenSet: TooltipTokenSet) {
+        // Update colors
+        let textColor = UIColor(dynamicColor: tokenSet[.textColor].dynamicColor)
+        backgroundView.backgroundColor = UIColor(dynamicColor: tokenSet[.tooltipColor].dynamicColor)
+        messageLabel.textColor = textColor
+        titleLabel?.textColor = textColor
+
+        // Update fonts
+        messageLabel.font = UIFont.fluent(tokenSet[.messageLabelTextStyle].fontInfo)
+        if let titleLabel = titleLabel {
+            titleLabel.font = UIFont.fluent(tokenSet[.titleLabelTextStyle].fontInfo)
+        }
+
+        // Update text container size
+        backgroundView.layer.cornerRadius = tokenSet[.backgroundCornerRadius].float
+        textContainer.frame = backgroundView.frame.insetBy(dx: TooltipTokenSet.paddingHorizontal, dy: (title != nil) ? TooltipTokenSet.paddingVerticalWithTitle : TooltipTokenSet.paddingVerticalWithoutTitle)
+        let preferredMessageSize = TooltipView.labelSizeThatFits(textContainer.frame.size,
+                                                                 text: message,
+                                                                 tokenSet: tokenSet,
+                                                                 isMessage: true)
+        messageLabel.frame.size = preferredMessageSize
+        if let titleLabel = titleLabel, let title = title {
+            let preferredTitleSize = TooltipView.labelSizeThatFits(textContainer.frame.size,
+                                                                   text: title,
+                                                                   tokenSet: tokenSet,
+                                                                   isMessage: false)
+            titleLabel.frame.size = preferredTitleSize
+            messageLabel.frame.origin = CGPoint(x: 0, y: titleLabel.frame.height + TooltipTokenSet.spacingVertical)
+        }
+
+        // Update tooltip size
+        positionController.updateArrowDirectionAndTooltipSize(for: message,
+                                                              title: title,
+                                                              tokenSet: tokenSet)
+        self.frame = positionController.tooltipRect
+
+        // Update shadows
+        updateShadows(tokenSet: tokenSet)
+    }
+
+    private func updateShadows(tokenSet: TooltipTokenSet) {
+        let backgroundCornerRadius = tokenSet[.backgroundCornerRadius].float
+        let shadowInfo = tokenSet[.shadowInfo].shadowInfo
+        let ambientShadow: CALayer = (layer.sublayers?[1])!
```

Comment:
we should not rely on force unwrap which give higher chance of crash during runtime.